cfme/tests/intelligence/reports/test_crud.py

```diff
@@ -80,6 +80,19 @@ def test_schedule_crud(schedule_data, appliance):
     schedule.delete()
 
 
+@pytest.mark.sauce
+@pytest.mark.tier(3)
+@test_requirements.report
+def test_reports_disable_enable_schedule(schedule_data, appliance):
+    schedules = ScheduleCollection(appliance)
```

Comment:
Collection items should be retrieved through the collections registered to the appliance.

```python
schedules = appliance.collections.schedules
```